### ISSUE TITLE
Bump outdated third_party/android_common/com.android.tools_sdk-common_25.0.0.jar to recent release

### ISSUE THREAD

**Issue by davido** [bazelbuild/bazel#11448]
This major Android dependency is totally outdated. Currently used version 25.0.0 in Bazel seems to be from April 2016. The current version is provided by Google and is from May 2020: 27.1.0-alpha09:

https://maven.google.com/com/android/tools/sdklib/27.1.0-alpha09/sdklib-27.1.0-alpha09.jar

Recent attempts to bump Guava dependency in Bazel are failing because of the collision of recent Guava releases and Guava version used in `tools_sdk-common_25.0.0.jar`.

See these discusions: [here ](https://github.com/bazelbuild/bazel/pull/10394#issuecomment-566784597) and [here](https://github.com/bazelbuild/bazel/pull/11426#issuecomment-630546492).

This [PR ](https://github.com/bazelbuild/bazel/pull/10394) and this [PR ](https://github.com/bazelbuild/bazel/pull/11426) are failing Android tests because of Guava version collision.

//CC @ahumesky @jin @tetromino

**Comment by aiuto** [comment]
There are 3 or 5 issues and PRs related to this, errorprone, and guava which all seem to to be related and fixable together. Can someone on the Android team answer decided a priority for dealing with it. If it is low priority for us, can we outline an acceptable plan for someone else to fix it.

**Comment by ahumesky** [comment]
Updating this dependency is quite tricky, and the long term plan is to separate this code from Bazel all together, but that will take some time. https://github.com/bazelbuild/bazel/pull/10394 and https://github.com/bazelbuild/bazel/pull/11426 were unblocked by patching this dependency, so I'll close this
